nav-challenge/src/components/pages/signUpPage/SignUpPage.tsx

```diff
@@ -1,8 +1,13 @@
 import { FC, useContext } from "react";
+
 import { SignUpFormProps } from "../../../utils/types/SignUpFormProps";
+
```

Comment:
Again, we can group the imports from our project and we should separate only our import groups by a newline.
By import groups I mean library import group, project import group and css import group